### PR TITLE
OSSMDOC-329: Performance scalability review

### PR DESCRIPTION
**Pull request by neal-timpe**
- Aligned team: Service mesh
- For branches: 4.6, 4.7, 4.8
- Moved a procedure module about how to update adjust the scalability. 
- Style updates
- https://issues.redhat.com/browse/OSSMDOC-329
- https://deploy-preview-32981--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-performance-scalability
- SME review: knrc
- QE review: yxun
- Peer review: mburke5678

### CODE REVIEW

**Comment by gbaufake** [comment]
@neal-timpe can you assign me as a reviewer on this one?

**Comment by neal-timpe** [comment]
@gbaufake This one is still in progress.  I've just started on it also. I don't think it's ready for an approval type review yet. 

But if you have feedback on this topic, it would be great if you could leave it here. I'll add you as a reviewer when I have more content.

**Review by knrc — APPROVED** [review]

**Review by knrc — APPROVED** [review]

**Review by yxun — APPROVED** [review]

File: modules/ossm-recommended-resources.adoc [hunk 1]
```diff
@@ -4,11 +4,26 @@ This module included in the following assemblies:
 ////
 
 [id="ossm-recommended-resources_{context}"]
-= Recommended compute resources
+= Setting limits on compute resources
 
-By default, `spec.proxy` has the settings `cpu:10m` and  `memory:128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
-The settings in the following example are based on 1,000 services and 1,000 requests per second. You can change the values for `cpu` and `memory` as necessary based on your environment.
+By default, `spec.proxy` has the settings `cpu: 10m` and  `memory: 128M`.  If you are using Pilot, `spec.runtime.components.pilot` has the same default values.
```
Comment (mburke5678) [inline modules/ossm-recommended-resources.adoc:9]: Is there an extra space between these two sentences?

File: modules/ossm-recommended-resources.adoc [hunk 2]
```diff
@@ -4,11 +4,26 @@ This module included in the following assemblies:
...
 
+The settings in the following example are based on 1,000 services and 1,000 requests per second. You can change the values for `cpu` and `memory` in the `ServiceMeshControlPlane`.
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. From the Project menu, select the project where you installed the control plane, for example `istio-system`.
```
Comment (mburke5678) [inline modules/ossm-recommended-resources.adoc:17]: For future consideration: Service mesh has it's own control plane, apart from the OpenShift control plane? Should this be mentioned somewhere, or are your users smart enough to know the difference and which control plane you are referencing?

File: service_mesh/v2x/ossm-deploy-production.adoc [hunk 3]
```diff
@@ -12,3 +12,7 @@ When you are ready to move from a basic installation to production, you must con
 * Test your configuration in a staging environment.
 
 include::modules/ossm-smcp-prod.adoc[leveloffset=+1]
+
+= Links
```
Comment (mburke5678) [inline service_mesh/v2x/ossm-deploy-production.adoc:16]: Do we use `= Links`? Not sure I've seen it. This shouldn't be `= Additional resources`?

File: service_mesh/v2x/ossm-performance-scalability.adoc [hunk 4]
```diff
@@ -1,13 +1,15 @@
 [id="ossm-performance-scalability"]
 = Performance and scalability
+include::modules/ossm-document-attributes.adoc[]
 :context: performance-scalability
 
 toc::[]
 
-The default `ServiceMeshControlPlane` settings are not intended for production use; they are designed to install successfully on a default OpenShift installation, which is an extremely resource limited environment. After you have verified a successful SMCP installation, you should modify the settings defined within the SMCP to suit your environment. This section provides guidance for configuring {ProductName} for production use.
+The default `ServiceMeshControlPlane` settings are not intended for production use; they are designed to install successfully on a default OpenShift installation, which is a resource limited environment. After you have verified a successful SMCP installation, you should modify the settings defined within the SMCP to suit your environment.
```
Comment (mburke5678) [inline service_mesh/v2x/ossm-performance-scalability.adoc:8]: I know this is existing text, but do you want to:
s/OpenShift/{product-title}
s/resource limited/resource-limited

File: modules/ossm-smcp-prod.adoc [hunk 5]
```diff
@@ -42,6 +42,8 @@ spec:
           resources: {}
 ----
 
+.. Configure the sampling rate for production. For more information, see the Performance and scalability section.
+
```
Comment (mburke5678) [inline modules/ossm-smcp-prod.adoc:46]: I know this is existing text, but I think you need a `+` between[ the _Verify the results._ steps and the command](https://github.com/openshift/openshift-docs/pull/32981/files#diff-4588b2ad0407c53d2acfa89719b3f18d6949c08e2684b962f3ae8827230043ecR49-R53).

**Comment by mburke5678** [comment]
A couple of nits (some for existing text) and one thought about a possible larger future change.

File: modules/ossm-recommended-resources.adoc (hunk 2 above)
Comment (neal-timpe) [reply modules/ossm-recommended-resources.adoc]: I added a name for the resource to clarify.

File: modules/ossm-recommended-resources.adoc (hunk 1 above)
Comment (neal-timpe) [reply modules/ossm-recommended-resources.adoc]: Not any more :)

File: service_mesh/v2x/ossm-deploy-production.adoc (hunk 3 above)
Comment (neal-timpe) [reply service_mesh/v2x/ossm-deploy-production.adoc]: fixed

File: modules/ossm-smcp-prod.adoc (hunk 5 above)
Comment (neal-timpe) [reply modules/ossm-smcp-prod.adoc:46]: fixed

File: service_mesh/v2x/ossm-performance-scalability.adoc (hunk 4 above)
Comment (neal-timpe) [reply service_mesh/v2x/ossm-performance-scalability.adoc]: fixed

**Comment by neal-timpe** [comment]
@mburke5678 I just knocked out your review comments. I'm ready for merge if you're the merger. 4.6-4.8 please?

**Pull request merged**